sdk/python/symbolchain/Transforms.py

```diff
@@ -0,0 +1,11 @@
+import hashlib
+
+import sha3
+
+
+def ripemd_keccak_256(payload):
+	"""Hashes payload with keccak 256 and then hashes the result with ripemd160."""
+
+	outer_hasher = hashlib.new('ripemd160')
+	outer_hasher.update(sha3.keccak_256(payload).digest())
```

Comment:
erm, not sure how I missed it, but this ofc is wrong 😬 

we'd need to do more or less what address is doing:
 * keccak
 * ripemd
 * concat proper pieces

Reply:
not enough ☕, I guess this is negligible given need to create rmd collision to achieve that